### PR TITLE
Add podspec files for Cocoapods support

### PR DESCRIPTION
**Pull request by ziyiz-amzn**
### Issue #, if available:
Add podspec files for Cocoapods support

#### Manual test cases (add more as needed):

- [ ] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [ ] Leave meeting
- [ ] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [ ] Send audio
- [ ] Receive audio
- [ ] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: AmazonChimeSDK-Bitcode.podspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  s.summary          = 'Amazon Chime SDK for iOS with Bitcode support.'
+  s.description      = 'An iOS client library for integrating multi-party communications powered by the Amazon Chime service.'
+  s.homepage         = 'https://github.com/aws/amazon-chime-sdk-ios'
+  s.license          = 'Apache License, Version 2.0'
+  s.author           = { 'Amazon Web Services' => 'amazonwebservices' }
+  s.source           = { :http => "https://amazon-chime-sdk-ios.s3.amazonaws.com/sdk/0.19.0/AmazonChimeSDK-0.19.0.tar.gz"}
+  s.ios.deployment_target = '10.0'
+  s.vendored_frameworks = "AmazonChimeSDK.xcframework"
+  s.swift_version    = '5.0'
+  s.dependency 'ZiyizSDKMedia-Bitcode', '~> 0.15.0'
```
Comment (hokyungh) [inline AmazonChimeSDK-Bitcode.podspec:13]: `ZiyizSDKMedia-Bitcode` doesn't sound right.

**Comment by hokyungh** [comment]
is there a way to just read from a file instead of hard coding version it to each file?

File: AmazonChimeSDK-Bitcode.podspec (hunk 1 above)
Comment (ziyiz-amzn) [reply AmazonChimeSDK-Bitcode.podspec:13]: good catch!

**Comment by ziyiz-amzn** [comment]
> 



> is there a way to just read from a file instead of hard coding version it to each file?

A script can do it. We will have the script to bump the versions.

**Review by georgezy-amzn — APPROVED** [review]

**Review by dingyishen-amazon — APPROVED** [review]

**Pull request merged**